.changeset/new-bugs-think.md

```diff
@@ -0,0 +1,7 @@
+---
+"@firebase/functions-compat": patch
+"@firebase/functions": patch
+"firebase-size-analysis": patch
```

Comment:
`firebase-size-analysis` can be removed, it's in the ignored changesets list. Actually, the change to that file seems unneeded (see below) so also for that reason.